synapse/lib/cell.py

```diff
@@ -89,6 +89,23 @@ def getCellIden(self):
 
         raise s_exc.AuthDeny(mesg='Caller must own task or be admin.', task=iden, user=str(self.user))
 
+    async def hivels(self, path=None):
```

Comment:
def cant expose direct hive access ( read or edit ) without some perms in play.  For now, check that the user is admin for these APIs.  we can figure out more granular perms soon.